package.json

```diff
@@ -24,7 +24,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node example/index.js",
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha test/**/*.spec.js test/renderers/**/*.spec.js"
```

Comment:
@rprieto they were, but int his PR I introduced special test suits that sole purpose is to generate samples for further testing in the main suite. Those tests are run programmatically (https://github.com/YemSalat/supersamples/blob/cbee6ff58fefb1092aac66c6b6d379f2a00f8a6f/test/renderers/postman/helper.js#L47)
I do not want mocha to pickup those tests and run them as part of the main suite.

I know my solution is not super-nice at the moment, will look into improving it before merge. Suggestions are very welcome!